Makefile

```diff
@@ -42,6 +42,10 @@ prepare-config:
 install-ods-project:
 	cd ods-setup && ./setup-ods-project.sh --namespace $(ODS_NAMESPACE) --reveal-secrets
 
+# AQUA SETUP
+## Setup central "ods" project.
+setup-aqua-configmap: install-ods-project
```

Comment:
I'd probably not use the dependency on `install-ods-project`.

Reply:
it was a suggestion for another team member but I do agree with you, those are topics that can be executed independently, I'll remove it. Thanks for your input